integration_tests/integration/notifications-settings.spec.ts

```diff
@@ -0,0 +1,25 @@
+import NotificationSettingsPage from '../pages/notificationSettings'
+import Page from '../pages/page'
+
+context('A staff member can view their notification settings', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubLogin')
+    cy.task('stubNotifications')
+    cy.task('stubNotificationCount')
+    cy.task('stubShifts')
+    cy.login()
+
+    cy.visit('/notifications/settings')
+  })
+
+  it('Notification page is visible', () => {
+    Page.verifyOnPage(NotificationSettingsPage)
```

Comment:
test of this page was missing altogether, although the jquery fix made no difference to the test